example_queries/configurations_insert_example.sql

```diff
@@ -0,0 +1,31 @@
+-- Insert configuration for both workcells
+INSERT INTO `biosero_uat`.`configurations` (
+  automation_system_id,
+  config_key,
+  config_value,
+  description,
+  created_at,
+  updated_at
+)
+VALUES
+((SELECT id FROM `biosero_uat`.`automation_systems` WHERE automation_system_name = 'CPA'), 'config_version', '2', 'version of these configurations', now(), now()),
+((SELECT id FROM `biosero_uat`.`automation_systems` WHERE automation_system_name = 'CPA'), 'change_description', 'moved the positive control', 'describes what was changed in this version', now(), now()),
```

Comment:
config_version_description?
config_version_changelog?

Reply:
I think we'll discuss these config key names with Biosero, use something that makes sense to them.